string group
string start_waypoint
---
---